auslib/test/admin/views/test_rules.py

```diff
@@ -295,6 +295,7 @@ def testGetRule(self):
             mig64=None,
         )
         self.assertEquals(json.loads(ret.data), expected)
+        self.assertIn('X-CSRF-Token', ret.headers)
```

Comment:
Nice additions to these tests - it's good that you're not changing the returned headers from either app as part of this.